includes/class-recaptcha.php

```diff
@@ -14,7 +14,7 @@
  */
 final class Recaptcha {
 	const SCRIPT_HANDLE  = 'newspack-recaptcha';
-	const THRESHOLD      = 0.5;
+	const THRESHOLD      = 0.25;
```

Comment:
Is this intended in this PR?

Reply:
Sorry, no. I was submitting so many test registrations that my submissions were starting to fail reCAPTCHA tests, so I had to bump down the threshold on my local site in order to continue testing. I guess that's a good sign it's working, though.